From 9728450429a0e0be4c2f09f2eae233f04fa0514c Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Wed, 11 Apr 2018 11:44:31 -0700
Subject: [PATCH] OS-6887 test-vm "test log archiving" failure: new console.log
 and platform.log surprised test case Reviewed by: Josh Wilsdon
 <josh@wilsdon.ca>

---
 src/vm/tests/test-docker.js | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/src/vm/tests/test-docker.js b/src/vm/tests/test-docker.js
index 8f2af6dd..19050933 100644
--- a/src/vm/tests/test-docker.js
+++ b/src/vm/tests/test-docker.js
@@ -1,4 +1,4 @@
-// Copyright 2017 Joyent, Inc.  All rights reserved.
+// Copyright (c) 2018, Joyent, Inc.
 //
 // These tests ensure that docker flag works as expected when setting/unsetting
 // Also test that /etc/resolv.conf, /etc/hosts and /etc/hostname are set
@@ -1080,8 +1080,10 @@ test('test log archiving', function (t) {
         }, function (cb) {
             var log_dir = '/zones/' + state.uuid + '/logs';
 
-            // ensure we have 2 log files now
+            // Ensure we have 2 "stdio.log*" files now.
             fs.readdir(log_dir, function (err, files) {
+                files = files.filter(
+                    function (f) { return f.indexOf('stdio.log') === 0; });
                 t.ok(files.length === 2, 'vm logs: ' + JSON.stringify(files));
                 cb();
             });
-- 
2.21.0

